core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -105,6 +106,7 @@
     this.projectId = matcher.group(1);
     this.regionId = matcher.group(3);
     this.instanceId = matcher.group(4);
+    this.prefixedInstanceId = String.format("%s~%s", this.regionId, this.instanceId);
```

Comment:
nit: "prefixed" isn't very specific 
```suggestion
    this.regionalizedInstanceId = String.format("%s~%s", this.regionId, this.instanceId);
```

Reply:
fixed